CMakeLists.txt

```diff
@@ -198,20 +199,14 @@ if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/CMAKELISTS.TXT")
         set(ln_name ${CMAKE_INSTALL_PREFIX}/${SHARE_DIR}/forward-models/res/${uc_job})
         INSTALL(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${ln_target}  ${ln_name})")
     endforeach()
+    foreach (script careful_copy_file copy_directory copy_file delete_directory delete_file make_directory move_file symlink make_symlink)
+        string(TOUPPER ${script} uc_scrip)
+        set(ln_target ${CMAKE_INSTALL_PREFIX}/${SHARE_DIR}/forward-models/shell/${script}_compatibility)
+        set(ln_name ${CMAKE_INSTALL_PREFIX}/${SHARE_DIR}/forward-models/shell/${uc_scrip})
+        INSTALL(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${ln_target}  ${ln_name})")
+    endforeach()
 endif()
 
-foreach (script ecl100 ecl300 flow rms)
-  install(PROGRAMS "share/ert/forward-models/res/script/${script}" DESTINATION "${SHARE_DIR}/forward-models/res/script")
-endforeach()
-
-foreach (script careful_copy_file copy_directory copy_file delete_directory delete_file make_directory move_file symlink)
-  install(PROGRAMS "share/ert/forward-models/shell/script/${script}" DESTINATION "${SHARE_DIR}/forward-models/shell/script")
-endforeach()
-
-foreach (script template_render)
-    install(PROGRAMS "share/ert/forward-models/templating/script/${script}" DESTINATION "${SHARE_DIR}/forward-models/templating/script")
-endforeach()
```

Comment:
This part has to stay in. I know the scripts are already copied by the `install(DIRECTORY ...)` further up, but installing them as `PROGRAMS` sets the permissions so that they are executable